Warn if acts_as_paranoid is called more than once on the same model

ignatiusreza opened the pull request:
which will breaks the method aliasing of `#destroy` and makes them lost
access to the original one defined by ActiveRecord

fix #489

ignatiusreza committed:
Warn if acts_as_paranoid is called more than once on the same model

which will breaks the method aliasing of `#destroy` and makes them lost
access to the original one defined by ActiveRecord

mathieujobin reviewed (approved):
This seems like a good idea... I wonder if my huge platform would be bitten by this...
I'd be temped to switch the `puts` for a raise if `Rails.env.test?` 
anyone else have an opinion on this?

I could include it in an upcoming release.
but needs version bump, changelog updated and rebased with `core` for tests to run.

ignatiusreza commented:
I rebased and added changelog

Given that paranoia is designed to be usable with activerecord outside of rails, I'm not quite sure how to best check for the rails environment. Any advice?

mathieujobin commented:
to answer your last question @ignatiusreza 

I recommend doing `if defined?(Rails)`
